sites/kit.svelte.dev/src/lib/docs/server/index.js

```diff
@@ -565,21 +566,27 @@ function convert_to_ts(js_code, indent = '', offset = '') {
 	return transformed === js_code ? undefined : transformed.replace(/\n\s*\n\s*\n/g, '\n\n');
 
 	/** @param {ts.JSDocTypeTag | ts.JSDocParameterTag} tag */
-	function get_type_name(tag) {
+	function get_type_info(tag) {
 		const type_text = tag.typeExpression.getText();
 		let name = type_text.slice(1, -1); // remove { }
 
-		const import_match = /import\('(.+?)'\)\.(\w+)/.exec(type_text);
+		const import_match = /import\('(.+?)'\)\.(\w+)(<{[\n\* \w:;]+}>)?/.exec(type_text);
```

Comment:
For simple same-line generics like `<Foo, Bar>` we need a comma here, too, and the opening/closing brackets would be gone, too
```suggestion
		const import_match = /import\('(.+?)'\)\.(\w+)(<{?[\n\* \w:;,]+}?>)?/.exec(type_text);
```